### DISCUSSION TITLE
Italian translation

### DISCUSSION THREAD

**Discussion by porcospine** [aclap-dev/video-downloadhelper#810] [Language & Translation]
Hi, updated the italian translation
[it.txt](https://github.com/user-attachments/files/15524979/it.txt)

**Comment by paulrouget** [comment]
Sorry we already got a translation for that language 🙏

I recommend that you try the upcoming version (v9.1 which should be released next week) see if the translation is correct.

If you want to keep track of the current translation status of your language, I recommend following this github repository: https://github.com/aclap-dev/vdhlocales
